src/internal/io.js

```diff
@@ -77,6 +77,9 @@ function getFnCallDesc(meth, fn, args) {
   } else if(fn.fn) {
     ({context, fn} = fn)
   }
+  if (context && is.string(fn)) {
+    fn = context[fn]
+  }
   check(fn, is.func, `${meth}: argument ${fn} is not a function`)
```

Comment:
yeah, i even think i had that thought but forgot to do something about it, will add this and merge in, thanks for the review